priv/catalog/222A57.yaml

```diff
@@ -16,10 +16,11 @@ remediation: |
 when: env.provider !in ["gcp", "aws"]
 
 facts:
-  - name: package_sbd
+  - name: compare_sbd_version
     gatherer: package_version
     argument: sbd,1.4.0
 
 expectations:
   - name: expectations_sbd_version
-    expect: facts.package_sbd < 1
+    expect: facts.compare_sbd_version < 1
+    failure_message: The installed SBD version is less than the recommended version (1.4.0)
```

Comment:
> What about `is older than` instead of `is less than`?

yes, thanks. That's indeed a better wording.

> 
> I am also wondering whether exposing the currently installed version would help the user thinking

yes, I thought about it too, but was not sure if adding an additional fact only to add information to the error message is really appreciated. If yes, I'm happy to change the checks in this regard.

Reply:
Not a blocker for me. Let's hear from @abravosuse and @jagabomb.

Reply:
@nelsonkopliku, I think your wording proposal make sense in terms of sounding better to the user. When it comes to exposing the current version it would be helpful. But if I think of the principle of the Checks, the reason it is failing is because the expected version is not being met. So I wonder if it is really worth it to expose the current version, since it would always fail. Maybe @abravosuse has a better opinion on this?

Reply:
I agree with the wording proposed by @nelsonkopliku. About the other topic, lets leave it AS IS for the time being and see what feedback we get from customers.